i18n/en.pot

```diff
@@ -69,6 +69,16 @@ msgstr "No preset option sets for this data item"
 msgid "Choose options"
 msgstr "Choose options"
 
+msgid "Previous selection was cleared"
+msgstr "Previous selection was cleared"
+
+msgid ""
+"Fetching details for the previously selected organisation unit failed. We "
```

Comment:
I don't think using the "we" form is part of our language in our apps? Suggest instead:
"The selection has been cleared so that a new selection can be made".